src/Kovan-DssSpell.sol

```diff
@@ -44,94 +75,218 @@ contract DssSpellAction is DssAction {
-        DssExecLib.authorize(MCD_VAT, MCD_CLIP_YFI_A);
-
+        DssExecLib.authorize(col.vat, col.clipper);
         // Authorize CLIP can access to DOG
-        DssExecLib.authorize(MCD_DOG, MCD_CLIP_YFI_A);
-
+        DssExecLib.authorize(col.dog, col.clipper);
         // Authorize DOG can kick auctions on CLIP
-        DssExecLib.authorize(MCD_CLIP_YFI_A, MCD_DOG);
-
-        // Authorize the new END to access the YFI CLIP
-        DssExecLib.authorize(MCD_CLIP_YFI_A, MCD_END);
-
+        DssExecLib.authorize(col.clipper, col.dog);
+        // Authorize the new END to access the ETH CLIP
+        DssExecLib.authorize(col.clipper, col.end);
         // Authorize CLIPPERMOM can set the stopped flag in CLIP
-        DssExecLib.authorize(MCD_CLIP_YFI_A, CLIPPER_MOM);
-
-        // Authorize new ESM to execute in YFI-A Clipper
-        DssExecLib.authorize(MCD_CLIP_YFI_A, MCD_ESM);
-
-        // Whitelist CLIP in the YFI osm
-        DssExecLib.addReaderToOSMWhitelist(PIP_YFI, MCD_CLIP_YFI_A);
-
-        // Whitelist CLIPPER_MOM in the YFI osm
-        DssExecLib.addReaderToOSMWhitelist(PIP_YFI, CLIPPER_MOM);
-
+        DssExecLib.authorize(col.clipper, col.clipperMom);
+        // Authorize new ESM to execute in ETH-A Clipper
+        DssExecLib.authorize(col.clipper, col.esm);
+        // Whitelist CLIP in the ETH osm
+        DssExecLib.addReaderToOSMWhitelist(col.pip, col.clipper);
+        // Whitelist clipperMom in the ETH osm
+        DssExecLib.addReaderToOSMWhitelist(col.pip, col.clipperMom);
         // No more auctions kicked via the CAT:
-        DssExecLib.deauthorize(MCD_FLIP_YFI_A, MCD_CAT);
-
-        // No more circuit breaker for the FLIP in YFI-A:
-        DssExecLib.deauthorize(MCD_FLIP_YFI_A, DssExecLib.flipperMom());
-
-        Fileable(MCD_DOG).file("YFI-A", "hole", 5_000 * RAD);
-        Fileable(MCD_DOG).file("YFI-A", "chop", 113 * WAD / 100);
-        Fileable(MCD_CLIP_YFI_A).file("buf", 130 * RAY / 100);
-        Fileable(MCD_CLIP_YFI_A).file("tail", 140 minutes);
-        Fileable(MCD_CLIP_YFI_A).file("cusp", 40 * RAY / 100);
-        Fileable(MCD_CLIP_YFI_A).file("chip", 1 * WAD / 1000);
-        Fileable(MCD_CLIP_YFI_A).file("tip", 0);
-        Fileable(MCD_CLIP_CALC_YFI_A).file("cut", 99 * RAY / 100); // 1% cut
-        Fileable(MCD_CLIP_CALC_YFI_A).file("step", 90 seconds);
-
+        DssExecLib.deauthorize(col.flipper, col.cat);
+        // No more circuit breaker for the FLIP in ETH-A:
+        DssExecLib.deauthorize(col.flipper, col.flipperMom);
+        // Set values
+        Fileable(col.dog).file(col.ilk, "hole", col.hole);
+        Fileable(col.dog).file(col.ilk, "chop", col.chop);
+        Fileable(col.clipper).file("buf", col.buf);
+        Fileable(col.clipper).file("tail", col.tail);
+        Fileable(col.clipper).file("cusp", col.cusp);
+        Fileable(col.clipper).file("chip", col.chip);
+        Fileable(col.clipper).file("tip", col.tip);
+        Fileable(col.calc).file("cut", col.cut); // 1% cut
```

Comment:
Seems like the `// 1% cut` comment should be removed here.

Reply:
Also, for safety, I'd add:
```
require(col.cut > 0, "DssSpell/abacus-zero-cut");
require(col.cut < RAY, "DssSpell/abacus-cut-gt-RAY");
```

Reply:
I'm removing the comment. Regarding the require's, shouldn't we add these type to all the values. Btw we have the tests where ranges are checked in general. We could add that one there.

Reply:
Sure, if you'd rather do it in the tests, that's fine as well.

Reply:
Done, haven't checked if the test is passing. The CI will say.